docs/src/templates/pages/documentation/general/gulp.hbs

```diff
@@ -1,169 +1,120 @@
+Gulp and its associated plugins can be managed via [NPM](https://www.npmjs.com/) or [Yarn](https://yarnpkg.com/en/) package managers. We recomemnd using Yarn, which is why all fozzie modules come with a `yarn.lock` file at their root.
 
-Gulp and Gulp plugins are managed via [NPM](https://www.npmjs.com/), the [Node.js](https://nodejs.org/) package manager.
 
-Here we document two ways of running Gulp tasks locally: using Visual Studio and it’s built-in tools, or running Gulp through the command line.
+<a name="gulp-fozzie"></a><h2 class="sg-sectionHeading">gulp-build-fozzie</h2>
 
-<hr class="sectionSplitter">
+We manage all of our gulp tasks across projects with a shared set of configurable tasks using the [`gulp-build-fozzie` module](https://www.npmjs.com/package/@justeat/gulp-build-fozzie).
 
-<a name="gulp-vs"></a>
-## Using Gulp in Visual Studio
+For details on how to use and figure this module, take a look at the [module's documentation](https://www.npmjs.com/package/@justeat/gulp-build-fozzie#setup).
 
-Here we document the steps needed to successfully run Gulp using Visual Studio 2013 and 2015.
 
-It is recommended that you upgrade to VS 2015 as the extension required to run Gulp tasks – Task Runner Explorer – comes baked in by default.
+<a name="gulp-setup"></a><h2 class="sg-sectionHeading">Getting up and running with Gulp</h2>
 
-### Setup steps
+Here we document two ways of running Gulp tasks locally: using Visual Studio and its built-in tools, or running Gulp through the command line.
 
-Future versions of Visual Studio (or the current enterprise )
+<a name="gulp-cmd"></a>
+#### Using Gulp on Command Line
 
-To get up and running with using Gulp in your setup of Visual Studio, just carry out the following steps:
+The first method of using Gulp is straight through the Command Line or Terminal.  To do this simply:
 
 1. **[Install Node.js](https://nodejs.org/)**
 
-	Node.js – and it’s bundled package manage npm – are needed to run Gulp tasks with VS 2013 or VS 2015 Professional.
-
-	Installing this should be as simple as going to the [Node.js website](https://nodejs.org/) and hitting the 'Install' button, which will take you through the install process.  Go with all the defaults throughout the install process and this should install Node and it’s package manager on your system.
+    Node.js – and its bundled package manager npm – are needed to run Gulp tasks with VS 2013 or VS 2015 Professional.
 
-2. **For users of VS 2013 only** – **[Install Task Runner Explorer](https://visualstudiogallery.msdn.microsoft.com/8e1b4368-4afb-467a-bc13-9650572db708)**
+    Installing this should be as simple as going to the [Node.js website](https://nodejs.org/) and hitting the 'Install' button, which will take you through the install process.  Go with all the defaults throughout the install process and this should install Node and its package manager on your system.
```

Comment:
There isn't an install button on the Node website, maybe expand this sentence a little by specifying that they should download and install the latest LTS version indicated on the homepage?

Reply:
Good catch – thought I'd get away with just copying this bit… 🙄